Src/WitsmlExplorer.Frontend/components/ContextMenus/LogObjectContextMenu.tsx

```diff
@@ -119,6 +119,15 @@ const LogObjectContextMenu = (props: ObjectContextMenuProps): React.ReactElement
     });
   };
 
+  const onClickCountLogData = async () => {
+    dispatchOperation({ type: OperationType.HideContextMenu });
+    const logReference: LogObject = checkedObjects[0];
+    const checkLogHeaderJob: CheckLogHeaderJob = { logReference };
+    const jobId = await JobService.orderJob(JobType.CountLogDataRows, checkLogHeaderJob);
+    const reportModalProps = { jobId };
+    dispatchOperation({ type: OperationType.DisplayModal, payload: <ReportModal {...reportModalProps} /> });
```

Comment:
Can you add a check to see if jobId is set before showing the ReportModal? If it fails ordering the job, the jobId will be null. If you then open the ReportModal, it will wait forever on the not-existing job to finish. This  can happen  when the user needs to re-authenticate, but hits cancel instead of signing in.

Just add `if (jobId)...` like in onClickCheckHeader

Reply:
fixed